flutter_vlc_player/android/src/main/java/software/solid/fluttervlcplayer/FlutterVlcPlayer.java

```diff
@@ -465,10 +468,7 @@ long getAudioDelay() {
     }
 
     void addAudioTrack(String url, boolean isNetworkUrl, boolean isSelected) {
-        if (isNetworkUrl)
-            mediaPlayer.addSlave(Media.Slave.Type.Audio, Uri.parse(url), isSelected);
-        else
-            mediaPlayer.addSlave(Media.Slave.Type.Audio, url, isSelected);
+        mediaPlayer.addSlave(Media.Slave.Type.Audio, Uri.parse(url), isSelected);
```

Comment:
Might as well remove the unnecessary `isNetworkUrl` argument above.

Reply:
fixed